### DISCUSSION TITLE
A review is required is wrong when multiple are required

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#101660] [Mobile] [answered]
GitHub Mobile beta for Android 
https://github.com/adoptium/containers/pull/487
![Screenshot_20240123-092208.png](https://github.com/community/community/assets/2119212/621db904-d7ef-4137-97c0-897a59415cc9)
Shows that [gdams ✅](https://github.com/adoptium/containers/pull/487#pullrequestreview-1838895896) but says a review is required. 

Compare on desktop where it says that
![Screenshot_20240123-092242~2.png](https://github.com/community/community/assets/2119212/ca4ed485-7cf1-4699-a7aa-760c0071bbf9)

> At least 2 approving reviews are required by reviewers with write access.

**Comment by eliperkins** [comment, marked as answer]
Hey @jsoref, thanks for this feedback! I can see how frustrating it would be to have the incorrect merge conditions shown in the mobile app, as compared to the web browser.

I've created an issue on our internal tracking repo to unify this messaging across platforms. I'll work with our engineering team to get this addressed, and follow back up with you when it's been shipped. Thanks for helping us improve GitHub Mobile! 🙇

**Reply by jsoref** [reply]
Thanks. I should note that the text is generally (not just on Mobile) a mess. It isn't that "reviews" are required, it's that "approvals" are required.

Using the word "review" adds zero value when the word "approval" could be used. As long as it's clear how someone goes about approving things.

To that point, it isn't clear how to approve things – I've seen people occasionally complain about this.

**Reply by jsoref** [reply]
This experience is still bad...

https://github.com/open-telemetry/opentelemetry.io/pull/6018


### Mobile app
![Screenshot_20250122-071756.png](https://github.com/user-attachments/assets/77cccea9-8a16-4980-82c7-5ad33b140634)

### Mobile browser
#### New merge experience
![Screenshot_20250122-072215.png](https://github.com/user-attachments/assets/23b1567c-63f0-4136-abcd-9b861397c6c9)

![Screenshot_20250122-072313.png](https://github.com/user-attachments/assets/415d1faa-6ebe-48cc-ad39-616d0c35f0cb)

This is almost ok with things collapsed:

> Merge info
Review required
Code owner review required by reviewers with write access.
…
Merging is blocked
Waiting on code owner review from open-telemetry/docs-approvers.

Note that [this new experience is a disaster too](https://github.com/orgs/community/discussions/143787) 

#### Classic merge experience 

![Screenshot_20250122-072451.png](https://github.com/user-attachments/assets/4e1fe91b-b7da-4428-a86d-5ea8b999fc89)

This did a reasonable job:

> Code owner review required
Waiting on code owner review from open-telemetry/docs-approvers.
